src/main/kotlin/no/nav/familie/ef/sak/oppgave/TilordnetRessursService.kt

```diff
@@ -74,25 +82,42 @@ class TilordnetRessursService(
             oppgavetyper,
         )
 
-    fun utledAnsvarligSaksbehandlerForOppgave(behandleSakOppgave: Oppgave?): SaksbehandlerDto {
-        val tilOrdnetRessurs = behandleSakOppgave?.tilordnetRessurs?.let { hentSaksbehandlerInfo(it) }
-        val rolle = utledSaksbehandlerRolle(behandleSakOppgave)
+    fun utledAnsvarligSaksbehandlerForOppgave(
+        behandlingId: UUID,
+        behandleSakOppgave: Oppgave?,
+    ): SaksbehandlerDto {
+        val rolle = utledSaksbehandlerRolle(behandlingId, behandleSakOppgave)
+
+        val tilordnetRessurs =
+            if (rolle == SaksbehandlerRolle.OPPGAVE_FINNES_IKKE_SANNSYNLIGVIS_INNLOGGET_SAKSBEHANDLER) {
+                hentSaksbehandlerInfo(SikkerhetContext.hentSaksbehandler())
+            } else {
+                behandleSakOppgave?.tilordnetRessurs?.let { hentSaksbehandlerInfo(it) }
+            }
 
         return SaksbehandlerDto(
-            etternavn = tilOrdnetRessurs?.etternavn ?: "",
-            fornavn = tilOrdnetRessurs?.fornavn ?: "",
+            etternavn = tilordnetRessurs?.etternavn ?: "",
+            fornavn = tilordnetRessurs?.fornavn ?: "",
             rolle = rolle,
         )
     }
 
     fun hentSaksbehandlerInfo(navIdent: String) = oppgaveClient.hentSaksbehandlerInfo(navIdent)
 
-    private fun utledSaksbehandlerRolle(oppgave: Oppgave?): SaksbehandlerRolle {
+    private fun utledSaksbehandlerRolle(
+        behandlingId: UUID,
+        oppgave: Oppgave?,
+    ): SaksbehandlerRolle {
         if (erUtviklerMedVeilderrolle()) {
             return SaksbehandlerRolle.UTVIKLER_MED_VEILDERROLLE
         }
 
         if (oppgave == null) {
+            val behandling = behandlingRepository.findByIdOrThrow(behandlingId)
+
+            if (behandling.steg == StegType.REVURDERING_ÅRSAK || behandling.steg == StegType.VILKÅR || behandling.steg == StegType.BEREGNE_YTELSE) {
```

Comment:
Hva med `SEND_TIL_BESLUTTER` ?

Reply:
Isåfall - er det stegene som har `tillattFor -> SAKSBEHANDLER` vi egentlig ønsker å sjekke?

Reply:
Har vi noen tilfeller hvor man kommer rett til steget `SEND_TIL_BESLUTTER` etter at man har opprettet en revurdering da? Hvis ikke, så ønsker vi ikke å returnere sannsynligvis saksbehandler da, nei